packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -366,6 +366,114 @@ describe("calcite-input-time-picker", () => {
       expect(await inputTimePicker.getProperty("value")).toBe("01:02");
       await assertDisplayedTime(page, "01:02 AM");
     });
+
+    describe("value", async () => {
+      it("directly setting value in calciteInputTimePickerChange event listener without calling event.preventDefault will not interfere with subsequent user-initiated change events (#12391)", async () => {
+        const page = await newE2EPage();
+        await page.setContent(`<calcite-input-time-picker value="14:30"></calcite-input-time-picker>`);
```

Comment:
Can you use the `html` formatting utility here so Prettier can format this? Applies to other new test markup.